eh_personality.c

```diff
@@ -7,6 +7,14 @@
 #include "class.h"
 #include "objcxx_eh.h"
 
+#if defined(__SEH__) && !defined(__USING_SJLJ_EXCEPTIONS__)
```

Comment:
The second of these is never defined.
```suggestion
#ifdef __SEH__
```

Reply:
I think I [copied those lines from libcxxabi](https://github.com/llvm/llvm-project/blob/cfe9ccbddd98b55e49e46bb40877ece6a47a7625/libcxxabi/src/cxa_personality.cpp#L25-L33), but I’d be fine with removing that second check if we can’t come up with a reason for having it.